Models/DataProcessor.cs

```diff
@@ -50,6 +50,8 @@ public class DataProcessor : ISaberMovementDataProcessor {
 				var beforeCutBottomPos = prevData.bottomPos;
 				var afterCutTopPos = newData.topPos;
 				var afterCutBottomPos = newData.bottomPos;
+
+				// Raycasts are somewhat expensive - Might make sense to reuse the SSRC's values with a transpiler
```

Comment:
I looked into it and it tended to be 3-5x slower. When reusing the values, it sometimes didn't get them for some reason but it reused 85-90% of SSRC's values.

Reply:
Weird - I guess your transpiler just stored the result in some static field that you then retrieved?

Reply:
It added the ray result into a dictionary (`Dictionary<SaberSwingRatingCounter, Dictionary<BladeMovementDataElement, Vector3>>`), `DataProcessor` then tries to get the result from it. It sounds very inefficient, I know, but not using the SSRC and `BladeMovementDataElement` to get the correct one, completely messed up the overswing.

Reply:
That's what I tried before, it tended to double the overswing.

Reply:
Hm, well, realisticly it shouldnt really matter, was just a possible optimization that came to mind